src/styles/utils/variables.scss

```
@@ -0,0 +1,6 @@
+$white: #FFF;
+$blue-accent: #00ACDC;
+$main-accent: #060B35;
+$secondary: #616070;
+$active-color: #00ACDC;
```

Why did you create two variables with the same values?
```suggestion
$blue-accent: #00ACDC;
$main-accent: #060B35;
$secondary: #616070;
```